system/src/Grav/Common/Twig.php

```diff
@@ -183,6 +183,10 @@ public function processPage(Page $item, $content = null)
                 $twig_vars['content'] = $content;
                 $template = $item->template() . TEMPLATE_EXT;
                 $output = $local_twig->render($template, $twig_vars);
```

Comment:
Oh, now I see it: you're using the same $output variable for two different meanings. Can you please change one of the variables to $modular_output (or just to be different) and add a comment why we need to do this? Otherwise someone comes later and tries to change the code without understanding it -- just breaking the feature again.